kickoff/model.py

```diff
@@ -254,10 +267,34 @@ def __init__(self, start, length, orderByDict):
         self.orderByDict = orderByDict
 
 
+def getReleasesListView(ready=None, complete=None,
+                        searchFilter=None, paginationCriteria=None):
+    filters = {}
+    if ready is not None:
+        filters['ready'] = ready
+    if complete is not None:
+        filters['complete'] = complete
+
+    if filters:
+        search_list = ProductReleasesView.getSearchList(filters)
+        qry = ProductReleasesView.query().filter(and_(*search_list))
+
+    if searchFilter:
+        search_list = ProductReleasesView.getSearchList(searchFilter)
+        qry = ProductReleasesView.query().filter(or_(*search_list))
+
+    orderByList = ProductReleasesView.getOrderByList(
+        paginationCriteria.orderByDict)
```

Comment:
The old implementation checks `paginationCriteria` before using it. The function signature tells me that it can be set to `None` which would fail run time. Probably better to require it to be passed explicitly.